js/index.js

```diff
@@ -0,0 +1,94 @@
+const btn = document.querySelector('#newbook');
+const bookForm = document.querySelector('#bookform');
+
+function Book() {
+  this.title = document.querySelector('#booktitle').value;
+  this.author = document.querySelector('#author').value;
+  this.read = document.querySelector('#read').checked;
+  this.pages = document.querySelector('#pages').value;
+}
```

Comment:
[OPTIONAL]
- Your app should have an array to store the instances of Book, please see the [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/library/book_array_requirement.png) from [Odin](https://www.theodinproject.com/courses/javascript/lessons/library) which has more information on how to use the array
**Even though this is marked as optional, I highly recommend you to work on this**